src/defaultConfig.ts

```diff
@@ -0,0 +1,11 @@
+import { ContextConfig } from "./types/contextConfig";
+import { FeaturesRegistry } from "./impl/featuresRegistry";
+import { EthTxBuilder } from "./impl/txBuilders/ethTxBuilder";
+import { GithubDappletProvider } from './impl/providers/githubDappletProvider';
+
+const DEFAULT_CONFIG: ContextConfig = {
+    provider: new GithubDappletProvider(),
+    features: new FeaturesRegistry([EthTxBuilder])
```

Comment:
> `features: new FeaturesRegistry([EthTxBuilder])`
`FeatureRegistry` should not be part of the configuration. It is the part of context implementation.

I would define only array of features in the config and would create a Registry in the context's constructor.
`features: [EthTxBuilder]`